DTS-STN/c19-benefits-node
# remove gst-benifit card from results.njk

# Description

remove gst benefit card 

## Test Instructions

1. run app
2. fill survey 
3. go to the end and ensure gst card is not there 

## Help Requested

- Things that you (the submitter)...
- want reviewers to pay very close...
- attention to when they review this PR.
- Note: If you want eng or non-engs (or both) help.

## Checklist
- [ x] Update CHANGELOG.md
